migrating/checklists/ossm-migrating-premigration-checklists-assembly.adoc

```diff
@@ -0,0 +1,257 @@
+// Module included in the following dir:
+//
+// * service-mesh-docs-main/checklist.adoc
+
+:_mod-docs-content-type: ASSEMBLY
+[id=ossm-migrating-premigration-checklists-assembly]
+= Premigration checklists
+include::_attributes/common-attributes.adoc[]
+:context: ossm-migrating-premigration-checklists-assembly
+
+toc::[]
+
+The checklists make the following assumptions:
+
+1. You have read "Important information to know before migrating" and understand the differences between {SMProduct} 2 and {SMProduct} 3.
+2. You have read through the migration recommendations.
+3. You want to migrate from {SMProduct} 2 to {SMProduct} 3.
+4. You are running {SMProduct} {SMv2Version}.
+5. You have upgraded your `ServiceMeshControlPlane` to the latest version.
+6. If you are using {KialiProduct}, you are running the latest version.
+7. You have installed the {SMProduct} Operator 3.
+
+To install the {SMProduct} 3 Operator, see: xref:../../install/ossm-installing-openshift-service-mesh.adoc[Installing OpenShift Service Mesh]
+
+[WARNING]
+====
+You must upgrade your {SMProduct} Operator 2 to the latest release **before** you install {SMProduct} Operator 3.
+
+If you upgrade {SMProduct} Operator 2 operator _after_ you install {SMProduct} Operator 3, you will need to uninstall and reinstall {SMProduct} Operator 3 to ensure the included custom resource definitions (CRDs) are up-to-date.
+====
```

Comment:
@FilipB and I have not been able to reproduce this so I'm not sure we need this warning anymore. @FilipB wdyt?

Reply:
Yes, I think the warning can be removed.  The need to have the latest 2.6.z operator is mentioned in multiple places anyway.